### PR TITLE
Adding a new BaseCardElement  Badge for iOS

### PR DESCRIPTION
**Pull request by reenudeswal**
# Related Issue

Adding support for new Element Badge

# Description

This PR implements a new `BaseCardElement`  `Badge`.
This new element supports following properties

image

https://github.com/user-attachments/assets/1b0ed25d-a25c-4546-81a5-73a81fb0a69c

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBadgeView.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//  Copyright © 2024 Microsoft. All rights reserved.
+//
+#import <UIKit/UIKit.h>
+#import "ACRView.h"
+#import "ACOEnums.h"
+#import "Badge.h"
+
+@interface ACRBadgeView : UIView
+
+- (instancetype)initWithRootView:(ACRView *)rootView
```
Comment (wyAbhishek) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBadgeView.h:15]: nit: Indentation looks off

**Comment by wyAbhishek** [comment]
I think, all new headers files should be in private headers directory. 
Also, you should be creating symlink in Include folder.

**Comment by reenudeswal** [comment]
> I think, all new headers files should be in private headers directory. Also, you should be creating symlink in Include folder.

fixed.

**Review by wyAbhishek — APPROVED** [review]

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBadgeView.h (hunk 1 above)
Comment (reenudeswal) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBadgeView.h:15]: fixed

File: ACRBadgeRenderer.h
Comment (wyAbhishek) [inline ACRBadgeRenderer.h:1]: This file is not needed.

File: ACRBadgeView.h
Comment (wyAbhishek) [inline ACRBadgeView.h:1]: This file is not needed

Comment (reenudeswal) [reply ACRBadgeView.h:1]: removed

File: ACRBadgeRenderer.h
Comment (reenudeswal) [reply ACRBadgeRenderer.h:1]: removed

**Pull request merged**